### PR TITLE
Fix for operator tests passing with no operator

### PR DESCRIPTION
**Pull request by edcdavid**

### CODE REVIEW

File: test-network-function/operator/suite.go [hunk 1]
```diff
@@ -70,6 +69,7 @@ var _ = ginkgo.Describe(testSpecName, func() {
 		env := config.GetTestEnvironment()
 		ginkgo.BeforeEach(func() {
 			env.LoadAndRefresh()
+			gomega.Expect(len(env.OperatorsUnderTest)).ToNot(gomega.Equal(0))
```
Comment (jun-rh) [inline test-network-function/operator/suite.go:72]: The intent is to fail the test if there is no operator found?

Comment (edcdavid) [reply test-network-function/operator/suite.go:72]: yes

**Review by wying3 — APPROVED** [review]

File: test-network-function/operator/suite.go (hunk 1 above)
Comment (HamadiSE) [reply test-network-function/operator/suite.go:72]: should we not just skip it ? Is operator deployment mandatory ?

Comment (jun-rh) [reply test-network-function/operator/suite.go:72]: exactly, similar to what we did in the scaling and node drain test, skip when no deployments are found.

Comment (edcdavid) [reply test-network-function/operator/suite.go]: done

**Review by HamadiSE — APPROVED** [review]

**Pull request merged**